Makefile

```diff
@@ -220,7 +220,7 @@ GOLANGCI_LINT_VERSION ?= v1.55.2
 GOIMPORTS_VERSION ?= v0.15.0
 ADDLICENSE_VERSION ?= v1.1.1
 # opm and operator-sdk version
-OP_VERSION ?= v1.33.0
+OP_VERSION ?= v1.36.0
```

Comment:
Does this version exist? I was not able to find it at https://github.com/operator-framework/operator-sdk/releases:

```
$ curl -sSL -i https://github.com/operator-framework/operator-sdk/releases/download/v1.36.0/operator-sdk_linux_amd64
HTTP/2 404 
server: GitHub.com
date: Wed, 21 Feb 2024 10:55:29 GMT
content-type: text/plain; charset=utf-8
vary: X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With
cache-control: no-cache
strict-transport-security: max-age=31536000; includeSubdomains; preload
x-frame-options: deny
x-content-type-options: nosniff
x-xss-protection: 0
referrer-policy: no-referrer-when-downgrade
content-security-policy: default-src 'none'; base-uri 'self'; connect-src 'self'; form-action 'self'; img-src 'self' data:; script-src 'self'; style-src 'unsafe-inline'
content-length: 9
x-github-request-id: 8946:3B50E7:55C14F:56A02C:65D5D6B6

Not Found                                                                                                                                                                                     
```

I suspect this is why CI is not passing on this PR: https://github.com/janus-idp/operator/actions/runs/7981629567/job/21808292116?pr=215#step:7:279
```
/home/runner/work/operator/operator/bin/golangci-lint run ./... --timeout 15m
Dowloading https://github.com/operator-framework/operator-sdk/releases/download/v1.36.0/operator-sdk_linux_amd64 to ./bin/operator-sdk
test -s /home/runner/work/operator/operator/bin/controller-gen || GOBIN=/home/runner/work/operator/operator/bin go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.11.3
/home/runner/work/operator/operator/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
test -s /home/runner/work/operator/operator/bin/kustomize || { curl -Ss "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash -s -- 3.8.7 /home/runner/work/operator/operator/bin; }
{Version:kustomize/v3.8.7 GitCommit:ad092cc7a91c07fdf63a2e4b7f13fa588a39af4f BuildDate:2020-11-11T23:14:14Z GoOs:linux GoArch:amd64}
kustomize installed to /home/runner/work/operator/operator/bin/kustomize
./bin/operator-sdk generate kustomize manifests -q
./bin/operator-sdk: line 1: Not: command not found
make: *** [Makefile:298: bundle] Error 127
Error: Process completed with exit code 2.
```

Reply:
@rm3l yeah, the versions don't line up.  I create separate environment variables